pkg/api/login_handler.go

```diff
@@ -19,17 +19,15 @@ type LoginResponseData struct {
 }
 
 const (
-	DefaultLoginExpiration          = 7 * 24 * time.Hour
-	DefaultInvitePasswordExpiration = 6 * time.Hour
-	DefaultResetPasswordExpiration  = 20 * time.Minute
+	DefaultLoginExpiration         = 7 * 24 * time.Hour
+	DefaultResetPasswordExpiration = 20 * time.Minute
 	// Deprecated: use gorilla managed session
 	JWTCookieName           = "access_token"
 	InternalAuthSessionName = "internal_auth_session"
 	TokenSessionKeyName     = "token"
 	OIDCAuthSessionName     = "oidc_auth_session"
 
-	LoginAudience         = "login"
-	ResetPasswordAudience = "reset_password"
+	LoginAudience = "login"
```

Comment:
The fact that the reset audience definition had to move is probably a sign that the login audience should also move over. (Not this PR, just an observation)